Ginger/GingerCoreCommon/SourceControlLib/SolutionInfo.cs

```diff
@@ -20,13 +20,6 @@ namespace GingerCoreNET.SourceControl
 {
     public class SolutionInfo
     {
-        public static partial class Fields
```

Comment:
Did you tested this source control classes shift with SVN and Git repositories?

Reply:
No Meni. I did not tested them. Do you know anyone using them in their projects?